src/libraries/Common/src/System/Security/Cryptography/Asn1/AsnXml.targets

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <AsnXml />
-    <_AsnXmlDiffCmd>%24%28command -v diff%29 -q -a</_AsnXmlDiffCmd>
+    <_AsnXmlDiffCmd>%24%28command -v git%29 diff --quiet</_AsnXmlDiffCmd>
```

Comment:
This change is due to the difference in tools that's available on CBL-Mariner?

Reply:
We explicitly support building tarball (source build) without git in arcade. Installing diffutils makes sense here, IMO, since it is available via base packages on other distros.

Reply:
The source-build build in dotnet/installer validates this behavior I think. In any case, RedHat and Caconical build the product in that mode (at least until the VMR comes online fully).